operating/default-security-policies.adoc

```diff
@@ -9,6 +9,15 @@ toc::[]
 [role="_abstract"]
 The default security policies in {product-title} provide broad coverage to identify security issues and ensure best practices for security in your environment. By configuring those policies, you can automatically prevent high-risk service deployments in your environment and respond to runtime security incidents.
 
+[NOTE]
+====
+The severity levels for policies in {product-title} are different from the severity levels that Red Hat Product Security assigns.
+
+The {product-title} policy severity levels are Critical, High, Medium, and Low. Red Hat Product Security rates vulnerability severity levels as Critical, Important, Moderate, and Low.
+
+While a policy's severity level and the Red Hat Product Security severity levels can interact, it's important to distinguish between them. For more information about the Red Hat Product Security severity levels, see link:https://access.redhat.com/security/updates/classification[Severity Ratings].
```

Comment:
```suggestion
While a policy's severity level and the Red Hat Product Security severity levels can interact, it is important to distinguish between them. For more information about the Red Hat Product Security severity levels, see link:https://access.redhat.com/security/updates/classification[Severity Ratings].
```

Reply:
Done. Thanks @jeana-redhat! When I wrote IBM content, contractions were okay to use, so that's one style tidbit that I  often forget about now. Old habits die hard!